src/libraries/System.Private.CoreLib/src/System/Buffer.cs

```diff
@@ -127,7 +127,7 @@ public static unsafe void MemoryCopy(void* source, void* destination, ulong dest
         // Non-inlinable wrapper around the QCall that avoids polluting the fast path
         // with P/Invoke prolog/epilog.
         [MethodImpl(MethodImplOptions.NoInlining)]
-        internal static unsafe void Memmove(ref byte dest, ref byte src, nuint len)
+        internal static unsafe void MemmoveInternal(ref byte dest, ref byte src, nuint len)
```

Comment:
Rename `ZeroMemory` to `ZeroMemoryInternal` for parity?

Reply:
Done.